spec/airport.spec.js

```diff
@@ -10,7 +10,7 @@ let plane;
 
 
 
-// Test 1:  AirportPlanes length increases to 1 when addPlane is called on an empty Airport with a Plane
+// Test 1: AirportPlanes length increases to 1 when addPlane is called on an empty Airport with a Plane
```

Comment:
I would make my test names more technical, for example I would say an empty Airport class, the addPlane function etc.